specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/Location.json

```diff
@@ -0,0 +1,234 @@
+        "title": "SubscriptionsManagementClient",
+        "description": "The Admin Subscriptions Management Client."
+    },
+    "host": "adminmanagement.local.azurestack.external",
+    "schemes": [
+        "https"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Subscriptions.Admin/locations": {
+            "get": {
+                "x-ms-examples": {
+                    "Get the list of usage aggregates for all tenants.": {
+                        "$ref": "./examples/Location/List.json"
+                    }
+                },
+                "description": "Get a list of all AzureStack location.",
+                "tags": [
+                    "Locations"
+                ],
+                "operationId": "Locations_List",
+                "parameters": [
+                    {
+                        "$ref": "Subscriptions.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "Subscriptions.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/LocationList"
+                        }
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": null
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Subscriptions.Admin/locations/{location}": {
+            "get": {
+                "x-ms-examples": {
+                    "Get the list of usage aggregates for all tenants.": {
+                        "$ref": "./examples/Location/Get.json"
+                    }
+                },
+                "description": "Get the specified location.",
+                "tags": [
+                    "Locations"
+                ],
+                "operationId": "Locations_Get",
+                "parameters": [
+                    {
+                        "$ref": "Subscriptions.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "Subscriptions.json#/parameters/LocationParameter"
+                    },
+                    {
+                        "$ref": "Subscriptions.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/Location"
+                        }
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": null
+                }
```

Comment:
This is not pageable

Reply:
yes, put result can never be pageable , it would be nice to have rule added to --azure-validator